http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/InstanceComponent.scala

```diff
@@ -107,7 +107,7 @@ object instanceQuery extends TableQuery(new InstanceTable(_)) {
       googleId = BigDecimal(instance.googleId),
       status = instance.status.entryName,
       ip = instance.ip.map(_.value),
-      dataprocRole = Some(instance.dataprocRole.entryName),
+      dataprocRole = Some(instance.dataprocRole.toString),
```

Comment:
Just checking, does this change the DB contract at all? We should keep the same string value if possible.

Reply:
it should be the same...but I'll double check

Reply:
```
scala> Master.toString
res1: String = Master

scala> Worker.toString
res2: String = Worker

scala> SecondaryWorker.toString
res3: String = SecondaryWorker
```

It matches what's in database currently